### PR TITLE
MGMT-7075: Requirements for ODF popover

### PR DESCRIPTION
**Pull request by jkilzi**
image

### CODE REVIEW

File: src/common/config/constants.ts [hunk 1]
```diff
@@ -10,7 +10,8 @@ export const OPENSHIFT_NETWORKING_DOCS_LINK =
   'https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal';
 export const CLUSTER_MANAGER_SITE_LINK = 'https://cloud.redhat.com/openshift/install/pull-secret';
 export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
-
+export const ODF_REQUIREMENTS_LINK =
+  'https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html-single/planning_your_deployment/index#resource-requirements_rhocs';
```
Comment (jkilzi) [inline src/common/config/constants.ts:14]:
```suggestion
  'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/planning_your_deployment/infrastructure-requirements_rhodf#resource-requirements_rhodf';
```

**Review by batzionb — DISMISSED** [review]

**Comment by nirfarkas** [comment]
LGTM

**Review by batzionb — APPROVED** [review]

File: src/common/config/constants.ts [hunk 2]
```diff
@@ -10,7 +10,8 @@ export const OPENSHIFT_NETWORKING_DOCS_LINK =
   'https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal';
 export const CLUSTER_MANAGER_SITE_LINK = 'https://cloud.redhat.com/openshift/install/pull-secret';
 export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
-
+export const ODF_REQUIREMENTS_LINK =
+  'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/planning_your_deployment/infrastructure-requirements_rhodf#resource-requirements_rhodf';
```
Comment (mareklibra) [inline src/common/config/constants.ts:14]: These links will be hard to maintain. If nothing else, then because of the OCP version.

Comment (mareklibra) [reply src/common/config/constants.ts:14]: Changing the version dynamically won't be reliable. We will need to remember reviewing the, occasionally. Would it make sense to add a task to jira? A repeated task? @jtomasek @jkilzi

**Review by mareklibra — APPROVED** [review]

**Comment by jkilzi** [comment]
Needs to update texts introduced in #1101 1101

File: src/common/config/constants.ts (hunk 2 above)
Comment (jkilzi) [reply src/common/config/constants.ts:14]: @mareklibra there's an epic in Jira to allow pointing to the OCP version the users choose to install https://issues.redhat.com/browse/MGMT-8534

**Review by batzionb — DISMISSED** [review]

**Pull request merged**